proto/access.proto

```diff
@@ -79,6 +79,12 @@ message AccessAttempt {
   // Actor describes the entity attempting to gain access.
   // Optional.
   Actor actor = 3;
+
+  // The time the access attempt was made.
+  google.protobuf.Timestamp time = 4;
+
+  // The name of the zone that was being accessed.
+  string zone = 5;
```

Comment:
Maybe use a different name to `zone` (as we use that elsewhere in sc-bos) or better describe what zone means here. Is it a sc-bos zone name, is it a human readable name representing the room, does it come from the device? etc.